busyorg/busy
# Fix displaying comment count element

### Changes

Summary of changes you made.

* Comment count element was visible event if there was no any comments. I've changed it to display it only when comments count is greater than 0.

### Test plan

You can use my [profile](https://staging.busy.org/@thequake) to check changes.

* Check and remember space between comment icon and share icon
* Run local version with this PR

### Demo

pre
![busy_pre](https://user-images.githubusercontent.com/4780115/39616707-14880c1e-4f7d-11e8-95e6-30d1b14aeb5d.png)


after
![busy_after](https://user-images.githubusercontent.com/4780115/39617283-606a4c8a-4f7f-11e8-9dac-c968614ae4cc.png)
